npm-shrinkwrap.json

```diff
@@ -4,13 +4,33 @@
   "lockfileVersion": 1,
   "requires": true,
   "dependencies": {
+    "@sinonjs/commons": {
```

Comment:
After npm-shrinkwrap.json changes, run `npm run clean-shrinkwrap` to fix it for older npm versions.

Reply:
DIdn't know about that !